src/main/java/org/sonarlint/intellij/core/SonarLintEngineFactory.java

```diff
@@ -170,6 +170,6 @@ private static Path getWorkDir() {
   }
 
   private Map<String, String> prepareExtraProps() {
-    return Collections.singletonMap("sonar.typescript.internal.typescriptLocation", application.getPluginPath().resolve("typescript").resolve("lib").toString());
+    return Collections.singletonMap("sonar.typescript.internal.typescriptLocation", application.getPluginPath().toString());
```

Comment:
@saberduck is it a safe change in the long run? Will the new plugin always support this path?

Reply:
I had a chat with @jblievremont , we simply take this path verbatim and set `NODE_PATH` . Ultimetely it's nodejs resolution algorithm which uses this path. It seems that indeed it should be parent of `typescript` directory

Reply:
Thanks!